recipes/medcoupling/LICENSE.txt

```diff
@@ -0,0 +1,458 @@
+                  GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
Is this file included in the distribution? if so, just refer to it directly.

Reply:
Unfortunately no. As far as I've seen it is only specified inside the top of the CMAKELists.txt file.

Reply:
The file is called `COPYING`

Reply:
see suggestion below and delete this file please.